### PR TITLE
Updated Management UI Helm Chart

### PR DESCRIPTION
**Pull request by werzl**
- Renamed chart to icap-management-ui
- Renamed service to icap-management-ui-service
- Updated description and version

### CODE REVIEW

File: administration/management-ui/values.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   port: 8080
 service:
-  name: icap-management-portal-service
+  name: icap-management-ui-service
   port: 80
 imagestore:
   icapmanagementui:
     repository: glasswallsolutions/icap-management-ui
-    tag: main
+    tag: main
```
Comment (mpigram) [inline administration/management-ui/values.yaml:11]: Need to add the latest main tag

**Review by mpigram — CHANGES_REQUESTED** [review]
See comment below mate

File: administration/management-ui/values.yaml (hunk 1 above)
Comment (werzl) [reply administration/management-ui/values.yaml:11]: main-latest?

Comment (mpigram) [reply administration/management-ui/values.yaml]: No the tag that has numbers in it 😅 looks like this main-f528704

**Pull request merged**